src/edit/items/ItemForm.js

```diff
@@ -204,34 +205,52 @@ class ItemForm extends React.Component {
     this.setState({ confirmTemporaryLocation, prevTemporaryLocation: prevTemporaryLoc });
   }
 
-  getActionMenu = ({ onToggle }) => {
-    const { onCancel } = this.props;
-    return (
-      <Button
-        data-test-inventory-cancel-item-edit-action
-        buttonStyle="dropdownItem"
-        onClick={() => {
-          onCancel();
-          onToggle();
-        }}
-      >
-        <Icon icon="times-circle">
-          <FormattedMessage id="ui-inventory.cancel" />
-        </Icon>
-      </Button>
-    );
-  };
-
   onSelectHandler = loc => this.selectTemporaryLocation(loc);
 
   setItemDamagedStatusDate = () => {
     this.props.form.change('itemDamagedStatusDate', new Date());
   }
 
-  render() {
+  getFooter = () => {
     const {
+      onCancel,
+      handleSubmit,
       pristine,
       submitting,
+      copy,
+    } = this.props;
+    const cancelButton = (
+      <Button
+        data-test-inventory-cancel-item-edit-action
+        buttonStyle="default mega"
+        id="cancel-item-edition"
```

Comment:
I see that the id was before `clickable-update-item`. If there is no problem changing that I suggest to name it as before for clarity.
```suggestion
        id="cancel-item-edit"
```

Reply:
thanks for suggested change! :)